src/infrastructure.sh

```diff
@@ -1292,6 +1292,44 @@ rlServiceRestore() {
     return $failed
 }
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# rlServiceRestore
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+: <<'=cut'
+=pod
+
+=head3 rlServiceStatus
+
+Print status of given C<service>.
+
+    rlServiceStatus service [service...]
+
```

Comment:
It's not clear what is the main purpose of the function:  status collection (`case $(rlServiceStatus)`), status checking (`if rlServiceStatus..`) or just logging?

In the first case, it should be also clear what keywords it exactly prints. In the second case, it's unclear what are the exit statuses.  Eg. is both "running" and "enabled" reported, and how exactly?

Reply:
Good point! My original intention was just to print in to the console and return the same exit code as `service/systemctl` would (status checking). When more services are in question the result is not really ideal for machine processing. We might do something about that. Any suggestion?
I'm not sure if status collection is necessary to have but it might help to process situation with more then one service.

Reply:
I've done some tweaks, please check it.